<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressBarWindow</class>
 <widget class="QDialog" name="ProgressBarWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>404</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>404</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>710</width>
    <height>404</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>461</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
	border-radius: 10px;
	background-color: rgb(220, 231, 255)
}

QProgressBar::chunk {
	border-radius: 10px;
	background-color: rgb(58, 68, 255)
}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="logoLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>461</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:whitesmoke;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;UnitedChanger&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
